app.js

```diff
@@ -38,7 +38,8 @@ class App {
 
     // Nunjucks configuration
     const appViews = [
-      path.join(__dirname, 'app/views/'),
+      path.join(__dirname, 'app/homepage'),
+      path.join(__dirname, 'app/views'),
```

Comment:
I think we can just have `app` as the base root and then each template is from this path

Reply:
I don't know what you mean

Reply:
we can just have the one path
`path.join(__dirname, 'app'),`
I don't think we should have to add more paths for each page.

Reply:
we need them both for now until all sections are done and all shared components come from the npm modules